### PR TITLE
swaggerize hammock

### PR DESCRIPTION
**Pull request by osher-stratoscale**
cli commands need to support swagger client generated responses

### CODE REVIEW

File: hammock/cli/command_manager.py [hunk 1]
```diff
@@ -41,21 +41,22 @@ def cli_command_map(func_name):
...
             resource_command_name
             if isinstance(resource_command_name, six.string_types)
             else names.to_command(instance.__class__.__name__)
         ]
-
         for name in dir(instance):
             # Get all clients' resources:
             attribute = getattr(instance, name)
-            if isinstance(attribute, type) or name.startswith('_') or name in IGNORES:
+            if isinstance(attribute, type) or name.startswith('_') or name in IGNORES or type(instance) == int:
```
Comment (lior-stratoscale) [inline hammock/cli/command_manager.py]: if isinstance(attribute, (type, int)) or name.startswith('_') or name in IGNORES:

File: hammock/cli/command_manager.py [hunk 2]
```diff
@@ -41,21 +41,22 @@ def cli_command_map(func_name):
...
                 continue
 
             if callable(attribute):
-                self._add_command(attribute, commands, cli_command_map(name))
+                try:
+                    self._add_command(attribute, commands, cli_command_map(name))
+                except:
+                    pass
```
Comment (lior-stratoscale) [inline hammock/cli/command_manager.py]: are you sure failing silently is the right thing to do?

File: hammock/cli/command_manager.py [hunk 3]
```diff
@@ -66,4 +67,7 @@ def _add_command(self, method, commands, command_name):
         command_type = command.factory(method, self.column_order, self.column_colors)
         command_name = ' '.join([part for part in commands if part])
         LOG.debug('Adding command: %s', command_name)
+        if '_with_http_info' in command_name:
```
Comment (lior-stratoscale) [inline hammock/cli/command_manager.py]: '_with_http_info' should be a constant with a *very* meaningful name

Comment (lior-stratoscale) [reply hammock/cli/command_manager.py]: plus probably the right thing to do is add this as additional metadata to the hammock decorators

File: hammock/cli/command.py [hunk 4]
```diff
@@ -132,6 +140,33 @@ def take_action(self, parsed_args):
...
 
+class SwaggerCommandList(Command, lister.Lister):
+    """
+    Command that returns a list of items,
+    each one is a dict.
+    """
+
+    def take_action(self, parsed_args):
+        objects = [munch.unmunchify(obj) for obj in self._action(parsed_args)]
+        # We expect the method to return a list of dicts, or a list of values.
```
Comment (lior-stratoscale) [inline hammock/cli/command.py]: what do you mean by 'values'? strings?

File: Makefile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-all: dist flake8 pylint unittest
+all: dist flake8 pylint #unittest
```
Comment (posener) [inline Makefile]: ?
please don't merge with this comment

**Pull request closed without merging**